tests/page-objects/forms/generic-form.wdio.page.js

```diff
@@ -0,0 +1,118 @@
+const nameField = () => $('#report-form form [name="/data/name"]');
+const submitButton = () => $('.enketo .submit');
+const submittedName = () => $('#reports-content .details ul li:first-child p');
+const formTitle = () => $('#form-title');
+const nextButton = () => $('button.btn.btn-primary.next-page');
+const fillForm = async (formFields) => {
+  for (const [fieldName] of Object.entries(formFields)) {
+    const field = formFields[fieldName];
+    const elm = $(field.css);
+    if (field.textField) {
+      await elm.sendKeys(field.value);
+    } else {
+      const radioButton = () => $(`${field.css}[value=${field.value}]`);
+      await radioButton.click();
+    }
+    if (field.endOfPage){
+      await module.exports.nextPageNative();
+    }
+  }
+};
+
+module.exports = {
+  fillForm,
+  submittedName,
+  submitButton,
+  nameField,
+  formTitle,
+  editForm: () => {
+    const editFormBtn = () => $('[href^="#/reports/edit"]>.fa-pencil');
+    editFormBtn.click();
+  },
+
+  editFormNative: async () => {
```

Comment:
There's a bit of duplication here. No need for the suffix of native anymore. After that there are two editForm functions. Can you remove the unnecessary one?